src/MarketSigner.js

```diff
@@ -402,6 +402,15 @@ function digestBid({
   });
 }
 
+function digestTransferBalanceToWeb3address({
+  owner, toWeb3Address, amount, fromCurrencyId, toCurrencyId, nonce,
+}) {
+  return concatHash({
+    types: ['string', 'string', 'string', 'string', 'uint8', 'uint8', 'uint32'],
```

Comment:
why are address not using the type 'address'? it is safer.

Reply:
ok, but this would require a refactor in our golang code. Do we want to do this?